### ISSUE TITLE
nomad-pack deploys do not "activate" action resources

### ISSUE THREAD

**Issue by attachmentgenie** [hashicorp/nomad-pack#495]
**Nomad Pack version**
Version in which this functionality is broken:

```
❯ nomad-pack version
Nomad Pack v0.1.0 (18c88d1)
```

**Issue**
While (re-)deploying a pack that has an action defined in one of its tasks, the associated "Actions" dropdown is never activated in the UI and the actions can not be triggered from the CLI.

**Reproduction steps**
1)  `nomad-pack generate pack nomad_actions`
2) `nomad-pack run nomad_actions`
3) modify the pack template, by adding an action

```
diff --git a/templates/nomad_actions.nomad.tpl b/templates/nomad_actions.nomad.tpl
index 7f5260c..7968449 100644
--- a/templates/nomad_actions.nomad.tpl
+++ b/templates/nomad_actions.nomad.tpl
@@ -46,6 +46,10 @@ job [[ template "job_name" . ]] {
       env {
         MESSAGE = [[ var "message" . | quote ]]
       }
+
+      action "whoami" {
+        command = "/bin/whoami"
+      }
     }
   }
 }
```
4) `nomad-pack run nomad_actions`
After this step the "Actions" dropdown is not visible and the action block is not in the job definition as displayed in the UI and trying to run the action from the CLI errrors out
```
❯ nomad action \              
  -group=app \        
  -task=server \
  -job=nomad_actions \
  whoami   
failed to exec into task: action whoami not found in task server
```
5) destroy the job and deploy again
Now the action resource is visible but the actions dropdown is not available and the cli still fails
6) Go to the job definition in the UI -> Edit -> Plan -> Run
```
+/- Job: "nomad_actions"
- Meta[pack.deployment_name]:	"nomad_actions"
- Meta[pack.job]:	"nomad_actions"
- Meta[pack.name]:	"nomad_actions"
- Meta[pack.path]:	"/Users/attachmentgenie/DevShed/StufisGeneralis/nomad_actions"
- Meta[pack.registry]:	"<<local folder>>"
- Meta[pack.version]:	"<<none>>"
+/- Task Group: "app" ( 1 create/destroy update )
+/- Task: "server" ( forces create/destroy update )
+ Action {
+ Command:	"/bin/whoami"
+ Name:	"whoami"}
```
7) After which the Actions dropdown is available and the action now succeeds on the CLI
```
❯ nomad action \ 
  -group=app \
  -task=server \
  -job=nomad_actions \
  whoami
root
```

F.Y.I:
`nomad-pack render nomad_actions/ -o .` -> `nomad run nomad_actions/nomad_actions.nomad` works as expected

**Comment by attachmentgenie** [comment]
seems to be fixed in the main already, 

`Nomad Pack v0.1.1-dev (4f6c84d)`

**Comment by attachmentgenie** [comment]
Fixed in #476 and released in 0.1.1
